plugins/aws/iam/sshKeysRotated.js

```diff
@@ -93,7 +93,7 @@ module.exports = {
         }
 
         if (!found) {
-            helpers.addResult(results, 0, 'No SSH keys found');
+            helpers.addResult(results, 0, 'No SSH keys found', 'global', obj.arn);
```

Comment:
you can't reference obj its out of scope.

Reply:
Oh yes you are right. In that case we shouldn't pass any resource to addResult.